code/modules/parallax/parallax_render_sources/parallax_render_source_groups.dm

```diff
@@ -106,6 +106,11 @@ Z_LEVEL_PARALLAX_RENDER_SOURCE_GROUP(5)
 		/atom/movable/screen/parallax_render_source/space_2,
 	)
 
+/datum/parallax_render_source_group/area/watchful_eye_sensor
+	parallax_render_source_types = list(	// these are placeholders until i upload better images
```

Comment:
?

Reply:
I wanted to get a high definition image of typhon's surface from cogwerks but life got in the way. I'll swap it out for void clouds and some asteroids